spec/requests/api/admin/search_references_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.describe Api::Admin::SearchReferencesController, type: :request do
```

Comment:
Missing request for SearchReferencesController in Admin.

Even if this is similar to the SearchReferencesController in V2,
those are in separate scopes, so I think a bit of duplication is acceptable.